docs/faq.rst

```diff
@@ -1,28 +1,93 @@
 Frequently Asked Questions
 **************************
 
-Installation issues
+General Questions
 ===================
 
-Bambi not working on Windows
+Why have a Bayesian regression library?
 ----------------------------
 
-If Bambi does not work on Windows you can...
+Bayesian modelling allows flexible (read 'bespoke') model specification and also provides an
+estimation of uncertainty in the model parameters. Both of these are wildly useful in 
+practice, in particular in a business context where the model is used to make decisions,
+and where a complex model may be needed to capture the underlying relationships. Further,
+Bayesian modelling allows graceful handling of small sample sizes by judicious use of
+prior distributions.
 
-* First
-* Second
 
 And finally, see `this issue <https://github.com/bambinos/bambi/issues/389>`_.
 
 
-Bambi not working on Linux
+I'm not familiar with Bayesian modelling, where can I learn more?
 ----------------------------
 
-* First
-* Second
+There are a number of excellent resources available online, including:
+    * [Bambi examples](https://bambinos.github.io/bambi/examples.html)
+    * [Bayesian Modeling and Computation](https://www.amazon.com/Bayesian-Modeling-Computation-Statistics-Science/dp/0124058884)
+    * [Bayesian Data Analysis](https://www.amazon.com/Bayesian-Analysis-Chapman-Statistical-Science/dp/1439840954)
+    * [Bayesian Methods for Hackers](https://github.com/CamDavidsonPilon/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers)
+
+What is the difference between Bambi and PyMC3?
+----------------------------
+    * Bambi is a wrapper around PyMC3, NumPyro, and BlackJax. It provides a simple interface for
```

Comment:
Yes, calling it a wrapper is incorrect. Thank you for the correction. I also took out mention of numpyro and blackjax here because they are mentioned as backends in multiple other questions.